src/flow-manager.c

```diff
@@ -264,6 +264,15 @@ static int FlowManagerFlowTimedOut(Flow *f, struct timeval *ts)
     }
 
     int server = 0, client = 0;
+
+    int state = SC_ATOMIC_GET(f->flow_state);
+    if (state == FLOW_STATE_BYPASSED)  {
+        if (FlowForceReassemblyNeedReassembly(f, &server, &client) == 1) {
```

Comment:
I did not get log after  30 sec. flow entry were only shown when shunting down suricata.